test/Npgsql.Tests/CommandTests.cs

```diff
@@ -904,30 +904,32 @@ public void ManyParameters()
         [Test, Description("Bypasses PostgreSQL's int16 limitation on the number of parameters")]
         [IssueLink("https://github.com/npgsql/npgsql/issues/831")]
         [IssueLink("https://github.com/npgsql/npgsql/issues/858")]
+        [IssueLink("https://github.com/npgsql/npgsql/issues/2703")]
         public void TooManyParameters()
         {
-            using (var conn = OpenConnection())
-            using (var cmd = new NpgsqlCommand { Connection = conn })
+            using var conn = OpenConnection();
+            using var cmd = new NpgsqlCommand { Connection = conn };
+            var sb = new StringBuilder("SOME RANDOM SQL ");
```

Comment:
Let's change it to `SELECT `. Otherwise, changes in the SQL parser on the server side could affect this test.

Reply:
The point is that this string is never supposed to get to the server, since we're supposed to block it at the client. In fact, a server-side error here is a clear indication that we're not doing our job correctly...

Reply:
@YohDeadfall does my explanation make sense, are you OK for merging?

Reply:
Yep.